master/reference/cni-plugin/configuration.md

```diff
@@ -280,6 +280,22 @@ There are two annotations to request a specific IP address:
         "cni.projectcalico.org/ipAddrsNoIpam": "[\"10.0.0.1\"]"
    ```
 
+   The CNI Plugin will not honor ipAddrsNoIpam requests unless it is enabled in the CNI config "feature_control" section:
```

Comment:
I think it's better to talk about this in terms of feature enablement rather than the CNI plugin respecting the annotation or not.

e.g. "This feature is disabled by default. It can be enabled in the `feature_control` section of the CNI network config."

We probably also want to stick a clear warning that without putting other checks in place, this feature allows for bypassing of network policy via IP spoofing and so if it's enabled the user should make sure the proper admission control is in place to prevent users from selecting arbitrary IP addresses.